katdal/datasources.py

```diff
@@ -635,19 +635,34 @@ def from_url(cls, url, chunk_store='auto', upgrade_flags=True, **kwargs):
             telstate = katsdptelstate.TelescopeState(katsdptelstate.memory.MemoryBackend())
             try:
                 telstate.load_from_file(url_parts.path)
-            except OSError as err:
-                raise DataSourceNotFound(str(err))
+            except OSError as e:
+                raise DataSourceNotFound(str(e))
```

Comment:
I'm guessing this is a left-over from your OSError foray, and can be reverted.

Reply:
No, I was just normalising the variable name for the exception with the rest of the file.